tox.ini

```diff
@@ -20,26 +20,26 @@ passenv =
     *
 usedevelop = false
 deps =
-    pytest==3.10.0
-    pytest-travis-fold==1.3.0
-    pytest-cov==2.6.0
-    PyYAML==5.1.2
+    pytest~=7.0.1
```

Comment:
Interested to know what the advantage is of using `~=` as a version classifier in the context of Tox. These aren't development dependencies, we're not forcing anyone to be on these versions except in the context of a Tox run, so surely it's better to pin them so we have reproducible behaviour?

Reply:
Yes, but also not expecting breaking changes on new patch versions of dev tools.

Reply:
Not totally convinced, but it's not a blocker 😄

Reply:
It's just better to use the freshest, but still compatible (as @expobrain mentioned) dependency e.g for security reasons.